src/sol/config.hpp

```diff
@@ -0,0 +1,53 @@
+// The MIT License (MIT)

```

Comment:
Prefer to use the system-installed sol if possible (optional at compile time). Or at least a subtree.